gsuite_activityevent_rules/gsuite_external_forwarding.yml

```diff
@@ -0,0 +1,75 @@
+AnalysisType: rule
+Filename: gsuite_external_forwarding.py
+RuleID: GSuite.ExternalMailForwarding
```

Comment:
Should this be added to a gsuite [pack](https://github.com/panther-labs/panther-analysis/blob/master/packs/gsuite_reports.yml#L2)? Also (out of scope), perhaps we should update the gsuite pack to apply to both report and activity event